src/App/VPSScan/Scan/ScotlandYard.hs

```diff
@@ -11,14 +11,22 @@ import App.VPSScan.Types
 import Control.Effect.Diagnostics
 import Control.Monad.IO.Class (MonadIO)
 import Data.Aeson
-import Data.Text (Text)
+import Data.Text (Text, append, pack)
 import Network.HTTP.Req
 import Prelude
 import App.Util (parseUri)
+import Data.Text.Encoding
+
+-- Prefix for Core's reverse proxy to SY
+coreProxyPrefix :: Url 'Https -> Url 'Https
+coreProxyPrefix baseurl = baseurl /: "api" /: "proxy" /: "scotland-yard"
+
+authHeader :: Text -> Option scheme
+authHeader apiKey = header "Authorization" (encodeUtf8 (append (pack "token ") apiKey))
```

Comment:
[nit]
- `append` on text is also called `<>`
- `pack "token "` is the same as `"token "` with the `OverloadedStrings` extension enabled
so:
```suggestion
authHeader apiKey = header "Authorization" (encodeUtf8 ("token " <> apiKey))
```